common/defaults.h

```diff
@@ -191,6 +192,10 @@ Default build-depended cvar and constant values
 	#define DEFAULT_M_IGNORE "0"
 #endif // DEFAULT_M_IGNORE
 
+#ifndef DEFAULT_JOY_DEADZONE
+	#define DEFAULT_JOY_DEADZONE "0"
```

Comment:
I think default can be 4096, so we have somewhat sane values for all platforms.

(and somewhere in the future get an UI for this)

Reply:
Ok, fixed